integration_tests/upgrade/add-new-node-after-upgrade.sh

```diff
@@ -96,10 +96,12 @@ check_adding_new_node() {
   test_divider
 
   echo "5. Start Node \"$node_name\""
-  docker exec -d $node_name cosmovisor run start
+  docker exec -d $node_name /var/lib/dcl/./node_helper.sh
 
   test_divider
 
+  sleep 1
```

Comment:
a bad practice actually. are you sure it would always work? can we instead do stick to some event to proceed with the remaining logic?